tests/components/screenlogic/test_config_flow.py

```diff
@@ -101,9 +101,33 @@ async def test_flow_discover_error(hass):
     assert result["errors"] == {}
     assert result["step_id"] == "gateway_entry"
 
+    with patch(
+        "homeassistant.components.screenlogic.config_flow.login.create_socket",
+        return_value=True,
+    ), patch(
+        "homeassistant.components.screenlogic.config_flow.login.gateway_connect",
+        return_value="00-C0-33-01-01-01",
+    ):
```

Comment:
Make sure to patch the integration setup functions on all create entry results to avoid setting up the whole integration.

Reply:
Commit https://github.com/home-assistant/core/pull/48136/commits/df0a10b963e7a75fe5cfbf2df1d25f68948de543